src/SimpleCalculator/Calculator.cs

```diff
@@ -2,15 +2,19 @@ namespace SimpleCalculator;
 
 public class Calculator
 {
-    public static void Main()
+    public static void Main(string[]args)
     {
+        Addition add = new Addition();
```

Comment:
all instances are created without even knowing if they are used or not. Could you please optimize it and create ONLY when it's used.

Reply:
Here, I used all the instances. So, I guess that I have to create them at the moment they will be used, right?

Reply:
correct. Basically you create instances and use some memory. It's not an issue for a small program, but for complicated one it will be mess and inefficient

Reply:
Oh, thank you so much. I understand